### DISCUSSION TITLE
Privacy policy

### DISCUSSION THREAD

**Discussion by ashishbarmase** [amidaware/tacticalrmm#1529] [General]
Hi, can someone please post the links to the Privacy policy? and to the security and compliance policy?  Please. I went through the documentation but not able to locate the same.

**Comment by dinger1986** [comment]
Does it say there is links for them?

**Comment by ashishbarmase** [comment]
I really don't know if there is link or not, but does this software has defined privacy policy? and the security and compliance policy? I can not try until I get approval from our security team and they need this info before approval.

**Reply by wh1te909** [reply]
https://license.tacticalrmm.com should cover everything. As far as privacy, this software is self hosted...you're in control of your own privacy. if you want us to host it for you that's a different story, please get in touch with [commercial support](https://support.amidaware.com/).
